### ISSUE TITLE
Get all of the software to update from a clean version of linux

### ISSUE THREAD

**Issue by Zacharyprime** [kipr/botui#79]
Right now I believe the only obstacle is getting botui to run on Qt5 in a stable way.

Once that is done I think we should be able to make a concise install method.

I would like to make it so that anyone can take a fresh install of linux (on a Pi) and follow a simple (copy and paste-able) set of instructions that would turn that linux install into a working Wombat with all of the software.

It would also be nice to include a shell script that does this in https://github.com/kipr/kipr-development-toolkit

This would make it so that we aren't just compounding issues with each new image file, and each image release could be done from a fresh start to ensure no snowballing has happened.

In addition or on a related note, it would be nice to dockerize (or other containerize) all of the software. It could even be a docker (or other container) package that contains all of the software. This would make the process described above even easier.
(I may create a new issue that addresses containerizing specifically)
